src/enums.rs

```diff
@@ -68,6 +53,8 @@ impl Into<c_int> for Dimensions {
 pub enum OutputDimension {
     TwoD,
     ThreeD,
+    #[cfg(feature = "v3_12_0")]
+    FourD,
```

Comment:
Was there a `OneD` added for this enum too maybe?

Reply:
This enum is used in the WKT writer, where 1 is not a valid value for the output dimension (only 2, 3, 4 are).